docs/how-to-install.md

```diff
@@ -145,11 +145,11 @@ For more detailed information about the Gravitino Iceberg REST server, please re
 
 ### Get the Apache Gravitino Docker image
 
-Gravitino publishes the Docker image to [Docker Hub](https://hub.docker.com/r/apache/gravitino/tags).
+Gravitino publishes the Docker image to [Docker Hub](https://hub.docker.com/r/datastrato/gravitino/tags).
```

Comment:
cc @yuqi1129

Reply:
We already moved docker hub to Apache, so it is not necessary to change back.

Reply:
Yes, the related PR has been merged into the main branch.